app/services/npq/dedupe_participant.rb

```diff
@@ -15,15 +15,30 @@ class DedupeParticipant
     def call
       return if invalid?
 
-      Identity::Transfer.call(from_user: npq_application.user, to_user: primary_user_for_trn)
+      if get_an_identity_id_clash?
+        # If both users have a get_an_identity_id, we transfer to the most recent
```

Comment:
I think this might cause an issue with flip flopping values as we call identity transfer in other places on primary trn users, let's chat tomorrow might be easier